modules/nodes-descheduler-configuring-interval.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-configuring-interval_{context}"]
+= Configuring the descheduler interval
+
+You can configure the amount of time between descheduler runs. The default is 3600 seconds (one hour).
+
+.Prerequisites
+* Cluster administrator privileges.
```

Comment:
Nit: new line between `.Prerequisites` and the first bullet point, for consistency.

Also, since `Cluster administrator privileges` isn't a full sentence, I don't think the period is needed.